nagios_check_resque.gemspec

```
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.0'
 
   spec.add_dependency 'nagios_check', '~> 0.4.0'
-  spec.add_dependency 'resque', '~> 1.25'
+  spec.add_dependency 'resque', '~> 2'
```

```suggestion
  spec.add_dependency 'resque', ['>= 1.25', '< 3']
```

No reason to drop support for Resque 1.x, though, right?

---

Here is the point :
Resque below version 2.0 would need Redis below version 4 (so redis 3.3.5)
However Resque version 2.0 cannot accept Redis below version 4.

You can see the change from Resque gem [here](https://github.com/resque/resque/commit/59a7d6190152b073c8fbecdfab1af54264a2ae1c) from version 1.27.4 to version 2.0.0

From my point of view, I don't see any reason to keep the support for Resque < 2.0 if we update the version of Nagios Check Resque.